components/private/Icons/icons/ic_more_horiz/src/MaterialIcons+ic_more_horiz.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+// Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Why 2016?

Reply:
I suspect that's when the script was written. I didn't modify that part in this PR.